libs/live-fns/azure-pipelines.js

```diff
@@ -0,0 +1,29 @@
+const got = require('../got.js')
+const cheerio = require('cheerio')
+
+module.exports = async (organization, project, definition, branch = 'master') => {
+  const response = await got(`https://dev.azure.com/${organization}/${project}/_apis/build/status/${definition}?branchName=${branch}`, { json: false })
+  const contentType = response.headers['content-type']
+
+  if (!contentType.includes('image/svg+xml')) {
+    return {
+      subject: 'Azure Pipelines',
+      status: 'unknown',
+      color: 'blue'
```

Comment:
`'grey'` would be better for unknown state

Reply:
Now it was fixed.